sdk/translation/Azure.AI.Translation.Text/README.md

```diff
@@ -65,6 +65,26 @@ string region = "<Text Translator Azure Region>";
 TextTranslationClient client = new TextTranslationClient(new AzureKeyCredential(apiKey), new Uri(endpoint), region);
 ```
 
+#### Create `TextTranslationClient` with Entra Credential
```

Comment:
Per the product branding guidelines, any occurrence of "Entra" should be prefixed with "Microsoft".

Reply:
Added & pushed.